packages/server/api/src/app/ee/platform-billing/platform-billing.service.ts

```diff
@@ -24,7 +24,7 @@ export const platformBillingService = (log: FastifyBaseLogger) => ({
         return platformBilling
     },
 
-    async update(platformId: string, tasksLimit: number | undefined, aiCreditsLimit: number | undefined): Promise<PlatformBilling> {
```

Comment:
nit: make this named constructor It would be easier to read when it's being called

{platformId, taskLimit} : {platformId: string, taskLimits: number | undefined}